common/persistence/tests/postgresql_test.go

```diff
@@ -194,34 +187,13 @@ FAIL: TestPostgreSQLQueuePersistence/TestNamespaceReplicationQueue (0.26s)
             	            	actual  : 98
             	Test:       	TestPostgreSQLQueuePersistence/TestNamespaceReplicationQueue
 */
-// func (p *PostgreSQLSuite) TestPostgreSQLQueuePersistence() {
+// func (p *PostgreSQLSuite) TestPostgreSQL12QueuePersistence() {
```

Comment:
what is the reason we keep commented out code?

Reply:
I honestly don't know. Based on the comments, it's a flaky test, so ideally it should be fixed, not removed.